docs/examples/vep.rst

```diff
@@ -0,0 +1,39 @@
+Variant Effect Predictor (VEP)
+==============================
+
+To use the `Ensembl Variant Effect Predictor <https://www.ensembl.org/vep>`_ with Hail on Google Dataproc,
+the ``--vep`` flag must be included when starting the cluster. Note that a cluster's VEP configuration is
+tied to a specific reference genome.
+
+.. code-block:: shell
+
+   hailctl dataproc start cluster-name --vep GRCh37 --requester-pays-allow-all --packages gnomad
+
+.. note::
+
+   VEP data is stored in requester pays buckets. Reading from these buckets will bill charges to the project
+   in which the cluster is created.
+
+Import variants into a sites-only Hail Table::
+
+   import hail as hl
+
+   ds = hl.import_vcf("/path/to/data.vcf.gz", reference_genome="GRCh37", drop_samples=True).rows()
+
+Annotate variants with VEP consequences::
+
+   from gnomad.utils.generic import vep_or_lookup_vep
+
+   ds = vep_or_lookup_vep(ds, reference="GRCh37", vep_config="gs://hail-us-vep/vep85-loftee-gcloud.json")
```

Comment:
Changes to the default VEP configuration in #197 have not yet been published to PyPI. Once they are, the `vep_config` argument can be removed here.